src/modm/platform/comp/stm32/base.hpp.in

```diff
@@ -58,6 +62,28 @@ namespace modm::platform
 		};
 	protected:
 		static constexpr uint32_t HysteresisMask = 0b11 << 16;
+	{% elif driver.type in ["stm32-tsmc90_g4_rockfish_cube"] -%}
```

Comment:
1. Why not `%%`?
2. `elif "g4" in driver.type`? Probably less fragile.

Reply:
> 1. Why not `%%`?

Consistency...